Dockerfile

```diff
@@ -17,10 +17,10 @@ RUN npm run build:prod
 # ones becase they use a different C compiler. Debian images also come with
 # all useful packages required for image manipulation out of the box. They
 # however weight a lot, approx. up to 1.5GiB per built image.
-FROM python:3.8 as production
+FROM python:3.11 as production
 
 ARG POETRY_INSTALL_ARGS="--no-dev"
-# IMPORTANT: Remember to review .circleci/config.yml when upgrading
+# IMPORTANT: Remember to review .github/workflows/lint.yaml and .github/workflows/test.yaml files when upgrading
 ARG POETRY_VERSION=1.7.1
```

Comment:
Can poetry be update please?

Reply:
It's also with the Wagtail 6.0 PR: https://github.com/torchbox/rca-wagtail-2019/pull/997/commits/e2270d8730169efb76e4cf751585c63780a5bb7b